.travis.yml

```diff
@@ -6,6 +6,7 @@ env:
   - VERSION=5 VARIANT=fpm-dev
   - VERSION=7 VARIANT=fpm
   - VERSION=7 VARIANT=fpm-dev
+  - VERSION=7 VARIANT=fpm-dev-mosaic
```

Comment:
can we just specify version instead?

Reply:
For MOSAIC we don't need eg. Node installed - thats why should create a separate image.

Regarding versioning, since for coma's image its version 7 (and actually php7.1 installed) - we should stick with current logic and change afterwards if needed.